pool/app/system_event/service_connections.ml

```diff
@@ -1,56 +1,37 @@
-open CCFun
-
 let name = "connection-watcher"
 let src = Logs.Src.create [%string "system_event.%{name}"]
 
-let verify_tenants () =
+let rerun_migrations_for_connection_issues () =
   let open Utils.Lwt_result.Infix in
   let open Database in
-  let open Status in
   Logs.debug ~src (fun m ->
-    m ~tags:(Logger.Tags.create root) "Check Connections");
-  let actions_by_status =
-    let to_connection_issue = flip Tenant.update_status ConnectionIssue in
-    let to_active = flip Tenant.update_status Active in
-    let clear_cache_event () =
```

Comment:
Aren't we now missing this event in the error middleware?

Reply:
fixed